docs/pages/blog/mui-x-v6-alpha-zero.md

```diff
@@ -0,0 +1,97 @@
+
+Next comes the beta phase, where the APIs will be more stable, and we'll focus on fixing bugs and polishing the hard edges. This phase is planned to take about one month.
+
+:::info
+Disclaimer: This timeline—2 months in alpha, 1 month in beta—is purely for reference.
+
+We operate in a dynamic environment that's subject to change, and the actual timeline could vary.
+We may need to do more iterations in the interest of delivering the best possible product.
+:::
+
+Finally, we’ll release the first `v6.0.0` stable, and we’ll continue improving the components and adding features, but without making any new breaking changes until the next major version.
+We hope to reach the stable release by the end of 2022.
+We’re establishing a one-year cycle for major versions, meaning the next major will arrive about one year after this release.
+
+## What happens to v5?
+
+During v6 pre-releases, v5 will continue to be the official current major, and it will remain supported during this time.
+However, we will only release new v5 versions as needed to patch bugs or add community contributions.
+
+All new features and enhancements will go to v6, and after its first stable release, v5 will officially transition to [long-term support](https://mui.com/versions/#long-term-support-lts) status.
+
+## Where's the v6 documentation?
+
+As mentioned above, v5 is still the official version, so by default, the documentation shows v5 features and API.
+The next version's documentation is in the `next` subdomain.
+
+- [https://next.mui.com/x/react-data-grid/](https://next.mui.com/x/react-data-grid/)
+- [https://next.mui.com/x/react-date-pickers/](https://next.mui.com/x/react-date-pickers/)
+
+## What’s coming next?
+
+The following is a list of enhancements in the pipeline for v6. It contains the highlights that will be included in the first versions.
+
+### Data Grid
+
+- **Extended customization abilities**.  
+   We want to empower more users to employ the Data Grid for complex use cases.
+
+  - [ApiRef in the community package](https://github.com/mui/mui-x/issues/6147).
+  - [Filtering on header](https://github.com/mui/mui-x/issues/6247).
+  - [Use the Data Grid internal components outside the grid](https://github.com/mui/mui-x/issues/2522).
+
+- **Improved look & feel**  
+   We’re polishing edges in terms of design and usability.
+
+  - [New column menu](https://github.com/mui/mui-x/issues/4929).
+  - [New column visibility panel](https://github.com/mui/mui-x/issues/5700).
+  - [Column Resize Bar only on hover](https://github.com/mui/mui-x/issues/1623).
+
+- **Rockstar feature**
+  - [Copy and paste from/to multiple cells](https://github.com/mui/mui-x/issues/199).
+
+### Date and Time Pickers
```

Comment:
Are we doing anything in terms of customizability? Like, expose a hook-only version or a headless version so we can also expose Joy UI version later. I mean, we could have these unstyled API exposed with `unstable_` so we can continue to iterate on it post v6.0.0 stable.

Reply:
> Are we doing changes in terms of improving customizability?

Yes! That's a good point I missed. 
But indeed the new date fields were built over hooks.

Reply:
We can communicate for the fields yes
For the pickers / views I would avoid to do it for now.